app/models/usage_update.rb

```diff
@@ -45,6 +61,32 @@ def self.get_data(report_url, _options = {})
     report
   end
 
+  def self.get_reports(options = {})
```

Comment:
Perhaps a better name is something like import_reports perhaps? The naming is misleading as it's called via UsageUpdateImportByYearJob which suggests updating, and here you're saying get.
A minor thing but could be clearer in case anybody tries to "get_reports" from a controller and ends up causing imports.

Reply:
sure that makes sense